### DISCUSSION TITLE
Does it allow to upgrade from one Win11 build to another on via WA on unsupported hardware?

### DISCUSSION THREAD

**Discussion by Anixx** [AveYo/MediaCreationTool.bat#332] [Q&A]
Or it is only for upgrading from Win10 to Win11?

**Comment by nondetect** [comment]
Yes it's can, use **Skip_TPM_Check_on_Dynamic_Update.cmd** from [here](https://github.com/AveYo/MediaCreationTool.bat/tree/main/bypass11) and you can update via WU from one build to another

**Reply by Anixx** [reply]
I have already tried this to upgrade 21h2 (22000). It does not find any updates. 
I had to download an image and then it upgraded in-place. No way via Windows Update.
